modules/nf-scil/registration/tractogram/main.nf

```diff
@@ -0,0 +1,83 @@
+process REGISTRATION_TRACTOGRAM {
```

Comment:
This module needs to change name. What it does is bundle registration, am I wrong ?

Reply:
I'll update the sheet, let's name this REGISTRATION_BUNDLEREGISTRATION.

Since to register tractograms, users should use #74 with the correct transforms, not this PR. Thanks a lot !